Source/Core/Core/HotkeyManager.cpp

```diff
@@ -437,3 +437,24 @@ void HotkeyManager::LoadDefaults(const ControllerInterface& ciface)
   set_key_expression(HK_UNDO_LOAD_STATE, NON + " & `F12`");
   set_key_expression(HK_UNDO_SAVE_STATE, SHIFT + " & `F12`");
 }
+
+// This override exists to load the old "Keys" group so pre-existing configs don't break.
```

Comment:
This and the TODO should likely go into the header, so it's more visible when looking at the interface.

LGTM otherwise.

Reply:
I'm in the process of reworking this PR, it's currently broken.